package.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "@pxblue/icons",
-    "version": "1.3.0",
+    "version": "1.4.0-beta.0",
```

Comment:
Does this mean the icons page and icons-svg will now be at a different version?

Reply:
No, we will keep them in sync even if it means publishing a new version without changes.

In this case we should be fine though, because we'll be updating them both with the pixel perfect fixes that Mariana did.

Reply:
I'm pretty sure this is not a breaking change (the existing font files that were generated are still generated and still work), which is why I only bumped the minor.